<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="de.bertelsmann.client.junit.tc245.J245_LivekasseInstoreRedemptionMonitoring_CancelRedemption" tests="2" skipped="0" failures="0" errors="0" timestamp="2025-03-05T11:45:42" hostname="EISMANN" time="5.723">
  <properties/>
  <testcase name="testMonitoringNachrichtInDatenbankUebernehmen_CancelRedemption_Fehler" classname="de.bertelsmann.client.junit.tc245.J245_LivekasseInstoreRedemptionMonitoring_CancelRedemption" time="3.582">
    <system-out><![CDATA[initializing... (initContext)class de.bertelsmann.client.junit.tc245.J245_LivekasseInstoreRedemptionMonitoring_CancelRedemption
initializing... (SetupOnce)class de.bertelsmann.client.junit.tc245.J245_LivekasseInstoreRedemptionMonitoring_CancelRedemption
initializing... (initContext)class de.bertelsmann.client.junit.tc245.J245_LivekasseInstoreRedemptionMonitoring_CancelRedemption
]]></system-out>
    <system-err><![CDATA[MÃ¤r 05, 2025 12:45:46 PM org.glassfish.jersey.test.inmemory.InMemoryTestContainerFactory$InMemoryTestContainer <init>
INFORMATION: Creating InMemoryTestContainer configured at the base URI http://localhost:9998/
]]></system-err>
  </testcase>
  <testcase name="testMonitoringNachrichtInDatenbankUebernehmen_CancelRedemption_Erfolgreich" classname="de.bertelsmann.client.junit.tc245.J245_LivekasseInstoreRedemptionMonitoring_CancelRedemption" time="2.141"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
